### ISSUE TITLE
How can I run multiple scenarios simultaneously when evaluating?

### ISSUE THREAD

**Issue by linglingxiansen** [ybgdgh/L3MVN#17]
I want to run multiple scenes or multiple episodes at the same time while evaluating, rather than one episode at a time. Can I configure this in args? I tried to change num_process, but it seems to be a parameter for the number of training processes. Thanks for your kindly help very much!

**Comment by linglingxiansen** [comment]
@ybgdgh Thanks very much for your help!  :)

**Comment by ybgdgh** [comment]
you can try to use "-n " to configure the parallel running.

**Comment by linglingxiansen** [comment]
> you can try to use "-n " to configure the parallel running.您可以尝试使用“-n”来配置并行运行。

Thanks very much. But when I tried to run it with "-n 6", it will run in parallel, but I only want to run 2000 episodes, and the program ends up running 16000, which doesn't seem to be running multiple episodes at the same time, but one episode running multiple times?

**Comment by linglingxiansen** [comment]
> you can try to use "-n " to configure the parallel running.

Thanks very much. But when I tried to run it with "-n 6", it will run in parallel, but I only want to run 2000 episodes, and the program ends up running 16000, which doesn't seem to be running multiple episodes at the same time, but one episode running multiple times?

**Comment by ybgdgh** [comment]
Yes you need to use "-n" and "--num_eval_episodes" together to make sure the final episode number. The total number is n*num_eval_episodes. For example, if you want to run 2000, you can set n = 5 and num_eval_episodes=400.

**Comment by linglingxiansen** [comment]
> Yes you need to use "-n" and "--num_eval_episodes" together to make sure the final episode number. The total number is n*num_eval_episodes. For example, if you want to run 2000, you can set n = 5 and num_eval_episodes=400.是的，您需要同时使用“-n”和“--num_eval_episodes”以确保最终的剧集编号。总数为 n*num_eval_episodes。例如，如果要运行 2000，可以设置 n = 5 和 num_eval_episodes=400。

Thanks very much! I got it, let me try. :)
